bokehjs/package-lock.json

```diff
@@ -3008,7 +3008,7 @@
     },
     "os-tmpdir": {
       "version": "1.0.2",
-      "resolved": "https://registry.npmjs.org/os-tmpdir/-/os-tmpdir-1.0.2.tgz",
+      "resolved": "http://registry.npmjs.org/os-tmpdir/-/os-tmpdir-1.0.2.tgz",
```

Comment:
Please revert this change. This is a side effect of npm being dubious piece of software. I suggest using `git add -p` (or `--patch`) to exclude unwanted changes.

Reply:
I hope I've succeeded.  I don't fully understand it but I definitely didn't want it there.

Reply:
Looks like it's gone now.  I'll read up on ```git add -p```.